### PR TITLE
* Fix scala format

### PR DESCRIPTION
**Pull request by ag-ramachandran**
The only changes are in the pom.xml file. The others are just format runs

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+align = none
+align.openParenDefnSite = false
+align.openParenCallSite = false
+align.tokens = []
+optIn = {
+  configStyleArguments = false
+}
+danglingParentheses.preset = false
+docstrings.style = Asterisk
+maxColumn = 98
```
Comment (ohadbitt) [inline .scalafmt.conf:27]: 98 is row length ? 120 or 160 are other langs standards

File: connector/pom.xml [hunk 2]
```diff
@@ -357,6 +357,16 @@
                 </executions>
                 <configuration>
                     <addScalacArgs>-target:jvm-1.8</addScalacArgs>
+                    <args>
+                        <arg>-Ywarn-unused</arg> <!-- if you need exactly RemoveUnused -->
```
Comment (ohadbitt) [inline connector/pom.xml:361]: what this means?

File: .scalafmt.conf [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+runner.dialect = scala212
+fileOverride {
+  "glob:**/src/**/scala-2.13/**.scala" {
```
Comment (ohadbitt) [inline .scalafmt.conf:30]: this is not our folders structures

**Review by ohadbitt — APPROVED** [review]
As you said - i only looked at the configuration not the files - if you didn't change anything there - go ahead

File: .scalafmt.conf (hunk 1 above)
Comment (ag-ramachandran) [reply .scalafmt.conf:27]: Thanks Ohad, fixed it to 120 chars.

File: connector/pom.xml (hunk 2 above)
Comment (ag-ramachandran) [reply connector/pom.xml:361]: If there are unused dependencies, this flag will report them. We eventually will add scalafix to fix code smells, we had to add this

File: .scalafmt.conf (hunk 3 above)
Comment (ag-ramachandran) [reply .scalafmt.conf:30]: Fixed this

**Pull request merged**